code/turf/floors.dm

```diff
@@ -1456,6 +1456,31 @@ TYPEINFO(/turf/simulated/floor/grass)
 /turf/proc/grassify()
 	.=0
 
+/// wetType: [-2 = glue, -1 = slime, 0 = dry, 1 = water, 2 = lube, 3 = superlube]
```

Comment:
```suggestion
/// wetType: [-2 = glue, -1 = slime, 0 = dry, 1 = water, 2 = lube, 3 = superlube]
/// silent: makes the overlay invisible and prevents the sound effect
```
Since it's not too clear otherwise what exactly `silent` does